config/unicorn.rb

```diff
@@ -0,0 +1,22 @@
+worker_processes Integer(ENV["WEB_CONCURRENCY"] || 3)
+timeout 15
+preload_app true
+
+before_fork do |server, worker|
+  Signal.trap 'TERM' do
+    puts 'Unicorn master intercepting TERM and sending myself QUIT instead'
+    Process.kill 'QUIT', Process.pid
+  end
+
+  defined?(ActiveRecord::Base) and
+  ActiveRecord::Base.connection.disconnect!
+end
+
+after_fork do |server, worker|
```

Comment:
Unused block argument - `server`. You can omit all the arguments if you don't care about them.<br>Unused block argument - `worker`. You can omit all the arguments if you don't care about them.